utils/camera.js

```diff
@@ -9,41 +22,57 @@ module.exports = {
     }
 
     const shakeTimer = game.time.create(false);
+    const shakeInterval = options.randomizeInterval? 
+      Math.random() * options.shakeInterval + options.shakeInterval : 
+      options.shakeInterval;
 
-    const shakeRange = 20;
-    const shakeInterval = 60;
-
-    let shakeCount = 10;
+    shakeTimer.loop(options.shakeInterval, () => {
+      if (options.shakeCount === 0) {
+        // if end shake reset camera, stop shake timer and call callback
+        if (target) {
+          camera.follow(target);
+        }
 
-    shakeTimer.loop(
-      shakeInterval,
-      function doShake() {
-        if (!shakeCount) {
-          if (target) {
-            camera.follow(target);
-          }
+        if (typeof callback === 'function') {
+          callback();
+        }
 
-          shakeTimer.stop();
+        shakeTimer.stop();
+        return;
+      }
 
-          if (typeof callback === 'function') {
-            callback();
-          }
+      // Calculate camera shift
+      let shift1, shift2;
+      const shakeRangeHalved = options.shakeRange / 2;
 
-          return;
+      if (options.randomShake) {
+        shift1 = game.rnd.integerInRange(-shakeRangeHalved, shakeRangeHalved);
+        shift2 = game.rnd.integerInRange(-shakeRangeHalved, shakeRangeHalved);
```

Comment:
Can we call these `shiftX` and `shiftY` instead, since they're always used that way?